product_uom_extra_data/migrations/11.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+xmlid_renames = [
```

Comment:
No need of this. Simply add the renaming of the module in apriori file in OpenUpgrade.

Reply:
Good